examples/dumpmodules.rs

```diff
@@ -6,8 +6,8 @@ use std::ffi::OsStr;
 use std::io::Write;
 use std::path::Path;
 
-extern crate minidump;
-extern crate minidump_common;
+
```

Comment:
I'm guessing you did a find-and-replace here but didn't remove the newlines? :sweat_smile:

Reply:
Guessed correctly. Shame on me, really.